src/components/InformCard.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+import {colors} from 'styles';
+import ProfileCircle from './ProfileCircle';
+
+type InfromCardProps = {
+  title: string;
+  address: string;
+  users?: string[] | undefined;
```

Comment:
```suggestion
import styled from 'styled-components';
import {colors} from 'styles';
import ProfileCircle from './ProfileCircle';

type InfromCardProps = {
  title: string;
  address: string;
  users: string[];
```
유저가 필수 인 상태로 변경되어야할 것 같아요!

Reply:
그리고, userProfile url 만 담고 있을 것 같아서,

userProfileImages 등과 같이 명시적이면 어떨까요?